### ISSUE TITLE
Python 3 import bug w/ oleid

### ISSUE THREAD

**Issue by dputtick** [decalage2/oletools#115]
When I build my project on Travis there are no problems in 2.7, but in 3.3+ I get an ImportError:

```
Traceback:
bin/filecheck.py:10: in <module>
    import oletools.oleid
../../../virtualenv/python3.5.2/lib/python3.5/site-packages/oletools/oleid.py:76: in <module>
    import thirdparty.olefile as olefile
E   ImportError: No module named 'thirdparty'
```

Looks like it might be a Python 3 import bug?

**Comment by decalage2** [comment]
Indeed Python 2 and 3 do not support relative imports the same way, I need to fix this in all oletools. (same issue as #101)

**Comment by dputtick** [comment]
Great, thanks! Looking forward to full python3 support :)

**Comment by decalage2** [comment]
This should now work with Python 2+3: @dputtick, could you please test it?

**Comment by dputtick** [comment]
It's fixed for me with oletools-0.51a1 in 2.7.12 and 3.5.2. Thanks 👍 . Will let you know if I find any other python3 compatibility issues
